pyproject.toml

```diff
@@ -18,15 +18,15 @@ packages = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6.1"
+python = "^3.8.0"
 pynetbox = "^5.0"
 toml = "^0.10"
 nornir = "^3.0"
 termcolor = "^1.1"
 click = "^7.1 || ^8.0"
 pydantic = "^1.6"
-genie = "~21.4.0"
-pyats = "~21.4.0"
+genie = "*"
```

Comment:
I think pinning protects us from undesired behavior, why we are losing it here?

Reply:
even we want to get rid of it, I would ping to something, even broader

Reply:
So you would like to see it still defined in the pyproject.toml instead of allowing the lock file to maintain this?